config/routes.rb

```diff
@@ -124,7 +125,22 @@
       resources :domain_blocks, only: :index, controller: :blocked_domains
     end
 
-    resource :two_factor_authentication, only: [:show, :create, :destroy]
+    resources :two_factor_authentication_methods, only: [:index] do
+      collection do
+        post 'disable'
```

Comment:
Code style consistency-wise, if there's no `/` in the path we use symbols like `post :disable`